docs/public/installation/zonemaster-gui.md

```diff
@@ -2,14 +2,34 @@
 
 ## Prerequisites
 
-Before installing Zonemaster Web GUI, you should [install Zonemaster::Engine
+Before installing Zonemaster-GUI, you should [install Zonemaster::Engine
 ][Zonemaster::Engine installation] and [Zonemaster::Backend][Zonemaster::Backend
 installation].
 
 Prerequisite for FreeBSD is that the package system is updated and activated,
 see FreeBSD section of [install Zonemaster::Engine][Zonemaster::Engine
 installation].
 
+## Upgrading or installation of custom built Zonemaster-GUI
+
+### Upgrading
+
+If this installation is an upgrade or reinstallation of Zonemaster-GUI and has
+done some configuration or customization, make sure to save updated file, e.g.
+`/var/www/html/zonemaster-web-gui/dist/config.json` and the `zonemaster.conf`
+Apache configuration file, before removing the old version. 
+
+You should also consider saving the old version under a new path instead of
+just removing it.
+
+Else upgrading is as simple as removing the old version and installing the new
+version in the same way as a new installation.
```

Comment:
When upgrading to a new major version you also need to consider the migration guide.